docs/html/index.html

```diff
@@ -1,23 +1,23 @@
 <!DOCTYPE html>
 <html>
 <head>
-<title>Welcome to nginx!</title>
+<title>Welcome to NGINX!</title>
 <style>
 html { color-scheme: light dark; }
 body { width: 35em; margin: 0 auto;
 font-family: Tahoma, Verdana, Arial, sans-serif; }
 </style>
 </head>
 <body>
-<h1>Welcome to nginx!</h1>
-<p>If you see this page, the nginx web server is successfully installed and
-working. Further configuration is required.</p>
+<h1>Welcome to NGINX!</h1>
+<p>If you see this page, NGINX is successfully installed and working.</p>
 
-<p>For online documentation and support please refer to
-<a href="http://nginx.org/">nginx.org</a>.<br/>
-Commercial support is available at
-<a href="http://nginx.com/">nginx.com</a>.</p>
+<p>If you have a suggestion, need help, or would like to contribute to our open source project, please visit <a href="https://github.com/nginx/nginx">github.com/nginx/nginx</a> or <a href="https://nginx.org/?welcome">nginx.org</a>.</p>
 
-<p><em>Thank you for using nginx.</em></p>
+<p>For online documentation and support please refer to <a href="https://nginx.org/en/docs/?welcome">nginx.org/en/docs/</a>.</p>
+
+<p>Enterprise distributions, commercial support, and training are available from F5, Inc at <a href="https://www.f5.com/products/nginx/?welcome">f5.com/products/nginx/</a></p>
```

Comment:
- Very long lines, please rewrap.
- `index.html` is used as an example to provide first steps, so it should be kept to minimum. Just updating the link from "nginx.com" is enough without adding advertisements (which are also out of scope).